static/src/javascripts/projects/common/modules/navigation/new-header.js

```diff
@@ -362,11 +348,59 @@ const trackRecentSearch = (): void => {
 
 const saveSearchTerm = (term: string) => local.set(SEARCH_STORAGE_KEY, term);
 
+const showMoreButton = (): void => {
+    fastdom
+        .read(() => {
+            const moreButton = document.querySelector('.js-show-more-button');
+            const subnav = document.querySelector('.js-expand-subnav');
+            const subnavList = document.querySelector(
+                '.js-get-last-child-subnav'
+            );
+
+            if (subnav && subnavList) {
+                const subnavItems = subnavList.querySelectorAll('li');
+                const lastChild = subnavItems[subnavItems.length - 1];
+
+                const lastChildRect = lastChild.getBoundingClientRect();
+                const subnavRect = subnav.getBoundingClientRect();
+
+                return { moreButton, lastChildRect, subnavRect };
+            }
+        })
+        .then(els => {
+            const { moreButton, lastChildRect, subnavRect } = els;
```

Comment:
You could just define `moreButton` in this block rather than passing it in the `els` argument as it's not used in the block it's currently defined in.

Reply:
But shouldn't it be in the fastdom read?

Reply:
Yes good point!